### PR TITLE
Update chapter 1, images for chapter 2

### PR DESCRIPTION
**Pull request by mikedotexe**
Added 7 helpful images, fixed issues brought up in the issue queue, added an image package

### CODE REVIEW

**Comment by mikedotexe** [comment]
This PR is good to review, friends. I may keep pushing updates because I need more resources to scale learning for my day-to-day tasks, but it doesn't mean this is a draft. 😇

File: docs/zero-to-hero/01-basics/03-hashing-and-unit-tests.md [hunk 1]
```diff
@@ -25,13 +25,38 @@ As mentioned in the first section of this **Basics** chapter, our smart contract
...
+    fn get_context(predecessor: AccountId) -> VMContextBuilder {
+        let mut builder = VMContextBuilder::new();
+        builder.predecessor_account_id(predecessor);
+        builder
+    }
+
+    #[test]
+    fn debug_get_hash() {
+        // Basic set up for a unit test
+        testing_env!(VMContextBuilder::new().build());
```
Comment (austinabell) [inline docs/zero-to-hero/01-basics/03-hashing-and-unit-tests.md:51]: Confused why get_context is required if not used?

**Review by austinabell — APPROVED** [review]
I like what these images add to the tutorial!

File: docs/zero-to-hero/01-basics/03-hashing-and-unit-tests.md (hunk 1 above)
Comment (mikedotexe) [reply docs/zero-to-hero/01-basics/03-hashing-and-unit-tests.md:51]: Good question. I saw a contributor ran into confusion, but perhaps I'll remove that function from this section and include it in the second unit test that actually uses it. Thanks

Comment (mikedotexe) [reply docs/zero-to-hero/01-basics/03-hashing-and-unit-tests.md:43]: Thanks again, Austin. I've pushed the update here and am much happier with it:
https://github.com/near/sdk-docs/pull/71/commits/56a9fd9afdf59187ad1b581d87e44b3cb9093889

**Pull request merged**